src/Makefile

```diff
@@ -65,20 +65,22 @@ ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
 ifeq ($(USE_POLLY_ACC),1)
-LLVMLINK += -lPollyPPCG
+LLVMLINK += -lPollyPPCG -lGPURuntime
 FLAGS += -DUSE_POLLY_ACC
-ifeq ($(CUDALIB_INCLUDE_DIR),)
-$(error CUDALIB_INCLUDE_DIR not set: If compiling with USE_POLLY_ACC, user must provide the path to CUDA header files)
-endif
-FLAGS += -I$(CUDALIB_INCLUDE_DIR)
-FLAGS += -I$(shell $(LLVM_CONFIG_HOST) --src-root)/tools/polly/tools
+FLAGS += -I$(shell $(LLVM_CONFIG_HOST) --src-root)/tools/polly/tools # Required to find GPURuntime/GPUJIT.h
 endif
 endif
 else
 SRCS += anticodegen
 LLVM_LIBS := support
 endif
 
+$(build_shlibdir)/libGPURuntime.$(SHLIB_EXT):
+ifeq ($(USE_POLLY_ACC), 1)
+# Place libGPURuntime such that it's accessible under libjulia's RPATH
+	@cp $(shell $(LLVM_CONFIG_HOST) --libdir)/libGPURuntime.$(SHLIB_EXT) $(build_shlibdir)
+endif
```

Comment:
The spirit of `USE_SYSTEM_XYZ` flags is to allow for users to use prebuilt binaries of external libraries.  E.g. to be able to run `USE_SYSTEM_LLVM=1` in order to use the LLVM libraries that you get when you `apt install llvm-dev`, or similar.  We don't really intend for it to be used with from-source builds that users create outside of the source tree.  This is because the number of possible states that users can get these out-of-source builds into is very large, and dealing with them all intelligently from within the Julia build system is nigh-impossible.  We'd much rather restrict the build system to handle only two cases; builds from within the Julia source tree, and re-using prebuilt binaries from system package managers.  As there are no system package managers that ship with the features that you're interested in at the moment, I would encourage you to do your development within the LLVM that is managed within the Julia build tree.  Any patches you are interested in adding to Julia to spice up the LLVM build recipes to get it built the way you want, I will be happy to review and merge, as this will make it a lot easier for others to replicate your work and build Julia with those features enabled.